src/modules/custom.rs

```diff
@@ -247,6 +248,15 @@ fn handle_powershell(command: &mut Command, shell: &str, shell_args: &[&str]) {
     }
 }
 
+#[cfg(windows)]
+fn handle_cmdexe(command: &mut Command, shell: &str, shell_args: &[&str]) {
+    let is_cmd = shell.ends_with("cmd.exe") || shell.ends_with("cmd");
+
+    if is_cmd && shell_args.is_empty() {
+        command.arg("/C");
+    }
+}
+
```

Comment:
I just tried this, and it doesn't seem to work with a simple custom module of just `echo test`. I think the issue is that the command gets passed via stdin. See also the discussion in #2624 and #2629.

Reply:
Does it work if you don't call this function at all (in `fn shell_command()`)? I will remove it then.

Reply:
I believe the output will include the prompt in that case. It might be best to just `log::error!` and return if the shell is detected as `cmd`.

Reply:
I think the reason the prompt gets shown is that cmd shell gets invoked without any params, i.e. `cmd echo test`. 

I have made the requested change.